addons/website/models/website_menu.py

```diff
@@ -26,6 +26,7 @@ def _default_sequence(self):
     child_id = fields.One2many('website.menu', 'parent_id', string='Child Menus')
     parent_path = fields.Char(index=True)
     is_visible = fields.Boolean(compute='_compute_visible', string='Is Visible')
+    mega_menu_content = fields.Html(string='Mega Menu Content', translate=True)
```

Comment:
- The string you specific is the default one (guessed from your variable name), so no need to define it.
- Check the other definitions of Html fields used in the website, the value for translate should not be `True` ;)